src/install/bin.zig

```diff
@@ -352,6 +355,184 @@ pub const Bin = extern struct {
             setPermissions(this.package_installed_node_modules, dest_path);
         }
 
+        const cmd_contents_begin =
```

Comment:
is there actually nothing we can do about this? ive always felt this is really big but at the same time, it is probably some crazy compatibility things